canbusload.c

```diff
@@ -325,6 +325,7 @@ int main(int argc, char **argv)
 		nptr = strchr(ptr, '@');
 
 		if (!nptr) {
+			fprintf(stderr, "CAN interface must have @ with bit rate\n");
```

Comment:
What about giving an example:

"Specify CAN interfaces in the form <CAN interface>@<bitrate>, e.g. can0@500000"